LICENSE.md

```diff
@@ -70,9 +71,9 @@ their own licenses:
 
 The following components of Julia's standard library have separate licenses:
 
-- base/fftw.jl (see [FFTW](http://fftw.org/doc/License-and-Copyright.html))
```

Comment:
The file itself is not and never has been GPL – it's MIT like the rest of Julia. So: ✔️